ou_dedetai/system.py

```diff
@@ -314,15 +314,36 @@ def get_superuser_command():
 
 
 def get_package_manager():
-    # Check for package manager and associated packages
-    if shutil.which('apt') is not None:  # debian, ubuntu
+    major_ver = distro.major_version()
+    logging.debug(f"{config.OS_NAME=}; {major_ver=}")
+    # Check for package manager and associated packages.
+    # NOTE: cabextract and sed are included in the appimage, so they have been
+    # removed from the dependencies lists.
+    if shutil.which('apt') is not None:  # debian, ubuntu, & derivatives
         config.PACKAGE_MANAGER_COMMAND_INSTALL = ["apt", "install", "-y"]
         config.PACKAGE_MANAGER_COMMAND_DOWNLOAD = ["apt", "install", "--download-only", "-y"]  # noqa: E501
         config.PACKAGE_MANAGER_COMMAND_REMOVE = ["apt", "remove", "-y"]
         config.PACKAGE_MANAGER_COMMAND_QUERY = ["dpkg", "-l"]
         config.QUERY_PREFIX = '.i  '
-        # NOTE: in 24.04 "p7zip-full" pkg is transitional toward "7zip"
-        config.PACKAGES = "binutils cabextract fuse3 p7zip-full wget winbind"
+        # Set default package list.
+        config.PACKAGES = (
+            "libfuse2 "  # appimages
+            "binutils wget winbind "  # wine
+            "p7zip-full "  # winetricks
+        )
+        # NOTE: Package names changed together for Ubuntu 24+, Debian 13+, and
+        # derivatives. This does not include an exhaustive list of distros that
+        # use 'apt', so others will have to be added as users report issues.
+        if (
+            (config.OS_NAME == 'ubuntu' and major_ver >= '24')
+            or (config.OS_NAME == 'debian' and major_ver >= '13')
+            or (config.OS_NAME == 'elementary' and major_ver >= '8')
```

Comment:
Linux Mint would be another we should add due to its popularity with new users

Reply:
done